packages/wonder-blocks-tabs/src/components/tabs-dropdown.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from "react";
+import {ActionItem, ActionMenu} from "@khanacademy/wonder-blocks-dropdown";
+import Button from "@khanacademy/wonder-blocks-button";
+import caretDown from "@phosphor-icons/core/bold/caret-down-bold.svg";
+import {StyleSheet} from "aphrodite";
+import {border, semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+import checkCircleIcon from "@phosphor-icons/core/fill/check-circle-fill.svg";
+import {PhosphorIcon} from "@khanacademy/wonder-blocks-icon";
+import {addStyle, StyleType} from "@khanacademy/wonder-blocks-core";
+
+const StyledDiv = addStyle("div");
+
+type TabDropdownItem = {
+    /**
+     * The label of the tab
+     */
+    label: string;
+    /**
+     * The id of the tab
+     */
+    id: string;
+    /**
+     * The contents for the tab
+     */
+    panel: React.ReactNode;
+};
+
+type Props = {
+    /**
+     * A unique id for the component.
+     */
+    id?: string;
+    /**
+     * Optional test ID for e2e testing.
+     */
+    testId?: string;
+    /**
+     * The tabs to render in the dropdown.
+     */
+    tabs: Array<TabDropdownItem>;
+    /**
+     * The id of the tab that is selected.
+     *
+     * If the selectedTabId is not valid, the `labels.defaultOpenerLabel` will
+     * be used to label the dropdown opener.
+     */
+    selectedTabId: string;
+    /**
+     * Called when a tab is selected.
+     */
+    onTabSelected: (id: string) => unknown;
```

Comment:
Even though the dropdown isn't really dealing with actual tabs anymore, I kept the naming similar to the Tabs component API so it's easier to use them both 

I started on the ResponsiveTabs implementation to make sure this approach is viable. It would look something like this: 

https://github.com/Khan/wonder-blocks/blob/75e3402493f008d803de71523c74bbba784c8379/packages/wonder-blocks-tabs/src/components/responsive-tabs.tsx#L91-L117